make/photon/clair/docker-entrypoint.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -e
 
-/harbor/install_cert.sh
-sudo -E -H -u \#10000 sh -c "/dumb-init -- /clair/clair -config /etc/clair/config.yaml $*"
+/home/clair/install_cert.sh
+/home/clair/dumb-init -- /home/clair/clair -config /etc/clair/config.yaml $*
```

Comment:
You have tested it right? not sure whether dumb-init does anything that requires higher privilege

Reply:
resolve it as it is tested